client/landing/stepper/declarative-flow/copy-site.tsx

```diff
@@ -52,7 +52,7 @@ const copySite: Flow = {
 						{ ...props }
 						title={ translate( 'We’re copying your site' ) }
 						subtitle={ translate(
-							'This may take a few minutes. Feel free to close this window and we will email you when the process is complete.'
+							'This may take a few minutes. Feel free to close this window, we’ll email you when it’s done.'
```

Comment:
TIL the difference between straight marks and smart marks: https://cmosshoptalk.com/2019/03/19/smart-apostrophes-cmos-6-117/